src/id_token.app.src

```diff
@@ -7,7 +7,8 @@
    [kernel,
     stdlib,
     hackney,
-    jose
+    jose,
+    base64url
```

Comment:
Don't forget to remove this if you follow my recommendation of using `jose_base64url`